sunpy/instr/goes.py

```diff
@@ -88,7 +100,8 @@ def get_goes_event_list(timerange, goes_class_filter=None):
 
     return goes_event_list
 
-def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH):
+def calculate_temperature_em(goeslc, abundances="coronal",
+                             download=False, download_dir=DATA_PATH):
     """
     Calculates and adds temperature and EM to a GOESLightCurve.
 
```

Comment:
This docstring is wrong, `goes_chianti_tem()` no longer exists under that name, and is not linked (which it can't be while it is not included in `__all__` and it would be weird having it in `__all__` when it is a 'private' function. If this is going to be the public interface to this functionality, this is the docstring which should do all the explaining.

As well as this, we are no longer returning a `GoesLightCurve`.